101-servicebus-create-namespace-geo-recoveryconfiguration/azuredeploy.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serviceBusNamespaceNamePrimary": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Service Bus namespace"
+      }
+    },
+    "serviceBusNamespaceNameSecondary": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Service Bus namespace"
+      }
+    },
+    "aliasName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Geo-Recovery Configuration Alias "
+      }
+    }
+  },
+  "variables": {
+    "defaultSASKeyName": "RootManageSharedAccessKey",
+    "defaultAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', parameters('serviceBusNamespaceNamePrimary'), variables('defaultSASKeyName'))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-04-01",
+      "name": "[parameters('serviceBusNamespaceNameSecondary')]",
+      "type": "Microsoft.ServiceBus/Namespaces",
+      "location": "[resourceGroup().location]",
```

Comment:
use parameters or variables for the locations - if you use params, do not include allowed values

Reply:
resolved